test/bundler-integration/environment/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "2"
+
+services:
+  bundler:
+    ports: ["3000:3000"]
+    image: accountabstraction/bundler:0.6.1
+    command: --network http://geth-dev:8545 --entryPoint 0x1E1EDc60c3C65E45C18D30a5D6E614c2e12d08E1 --show-stack-traces
```

Comment:
which entry point address is this?
0x1E1EDc60c3C65E45C18D30a5D6E614c2e12d08E1

Reply:
The address of the deployed entrypoint after calling https://github.com/bcnmy/scw-contracts/blob/test-bundler-integration/test/bundler-integration/environment/deployEntrypoint.ts, and the same as the the address of the entrypoint deployed in unit tests

Reply:
if it's not deterministic, is there some way we don't have to hardcode?

Reply:
Yeah I just pushed a fix for that, testing it currently